guides/browser-support.md

```diff
@@ -114,6 +114,37 @@ browsers:
 - iOS Safari
 - Chrome for Android
 
+#### Manual testing
+
+We use a mix of [Sauce Labs](https://saucelabs.com/) and
+[localtunnel](https://localtunnel.github.io/www/) to manually test our site.
+Be sure to review both the
+[testing](https://github.com/cfpb/development/blob/master/guides/testing.md)
+and [accessibility](https://github.com/cfpb/development/blob/master/guides/accessibility.md)
+guides when reviewing sites locally.
+
+##### Sauce Labs
+
+See [#171](https://github.com/cfpb/development/pull/171) for details on testing
```

Comment:
This should link to the Pull Request and not the document the 171 added?

Reply:
The PR isn't merged yet :( When it is, I'll update to point to that doc